discovery/sync_manager_test.go

```diff
@@ -685,3 +688,167 @@ func assertActiveSyncerTransition(t *testing.T, s *GossipSyncer, peer *mockPeer)
 	})
 	assertSyncerStatus(t, s, chansSynced, PassiveSync)
 }
+
+// TestSizeableMessage is a test implementation of lnwire.SizeableMessage.
+type TestSizeableMessage struct {
```

Comment:
why did you not use a mock ?

Reply:
Just needed some simpler behavior so I didn't reach for it 🤷